tests/test_prism.py

```diff
@@ -1393,7 +1393,7 @@ def test_prism_local_files_format_multiple_extensions(monkeypatch):
                                 "local_file_col_name": {
                                     "merge_pointer": "artifact",
                                     "gcs_uri_format": {
-                                        "format": "lambda val, ctx: 'subfolder/'+ctx['record']+'/artifact.'+val.rsplit('.', 1)[-1]",
+                                        "format": "lambda val, ctx: 'subfolder/' + ctx['record'] + '/artifact.'.format_map(ctx) + val.rsplit('.', 1)[-1]",
```

Comment:
no need for `.format_map` here either
```suggestion
                                        "format": "lambda val, ctx: 'subfolder/' + ctx['record'] + '/artifact.' + val.rsplit('.', 1)[-1]",
```